util/topgen.py

```diff
@@ -993,7 +879,7 @@ def main():
 
     for ip in generated_list:
         log.info("Appending {}".format(ip))
-        if ip == 'clkmgr':
+        if ip == 'clkmgr' or pass_idx:
```

Comment:
Let's explicitly add checker `pass_idx == 1` ? or I think it is more like `pass_idx != 0`

Reply:
yeah it's meant to be > 0.